indico/modules/rb_new/client/js/modules/bookRoom/actions.js

```diff
@@ -192,10 +195,22 @@ export const {searchRooms} = roomSearchActionsFactory('bookRoom');
 export const openBookRoom = (roomId, data = null) => modalActions.openModal('book-room', roomId, data);
 export const openUnavailableRooms = () => modalActions.openModal('unavailable-rooms');
 export const openBookingForm = (roomId, data) => modalActions.openModal('booking-form', roomId, data);
-export function setDate(date) {
+export function setTimelineDate(date) {
     return {type: SET_TIMELINE_DATE, date};
 }
 
 export function setTimelineMode(mode) {
     return {type: SET_TIMELINE_MODE, mode};
 }
+
+export function setUnavailableDate(date) {
```

Comment:
I would call those setUnavaiableNavDate, setUnavailableNavMode, etc... as a way to clarify what they do. Otherwise, it may look like the bookRoom page has some kind of 'unavailable mode'.